spec/checkout_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'checkout'
+
+describe Checkout do
+  let(:user_prompt) { "Please enter your phone number to complete the order:\n" + "+44 " }
+  let(:confirmation_prompt) { "Your order is on it's way! Please check your phone for confirmation.\n" }
+
+  describe '#place_order' do
+    it { is_expected.to respond_to(:place_order) }
+
+    it 'prompts the user for their phone number' do
+      allow(subject).to receive(:gets).and_return("1234567890")
```

Comment:
This stubbing for the user input looks great!

Reply:
Thanks for the input! Would you say generally this was the correct way to test an app that has so much user input? Or is there a better framework for it that would allow better test coverage?

Reply:
This is the way that I've seen it done in the past. I'm not aware of any frameworks off the top of my head